src/components/ui/use-toast.ts

```diff
@@ -17,12 +17,14 @@ type ToasterToast = ToastProps & {
   position?: string
 }
 
+/* eslint-disable */
```

Comment:
why?

Reply:
because there was generating an error and since it was done by shadcn or radix-ui, I don't think we should bother with those. Let me know if you think we should.

Reply:
ok, it is in the `ui` folder. Doesn't the ignore config already covers that?

Reply:
Oh right! I added that eslint disable comment before I knew I could add an ignore array in the eslint config and completely forgot about that, thank you!

I will remove those.

Reply:
I've now removed those eslint-disables from the files inside the `components/ui` since, as you correctly pointed out, they already are inside the `ignore` directives.